pkg/scheduler/eventhandlers_test.go

```diff
@@ -268,17 +275,15 @@ func TestPreCheckForNode(t *testing.T) {
 				node := st.MakeNode().Name("fake-node").Obj()
 				node.Spec.Taints = []v1.Taint{
 					{Key: "foo", Effect: v1.TaintEffectNoSchedule},
-					{Key: "bar", Effect: v1.TaintEffectPreferNoSchedule},
```

Comment:
Revert this change as it shouldn't impact the test.

Reply:
Sorry i was careless